prov/rxm/src/rxm_ep.c

```diff
@@ -644,6 +643,9 @@ rxm_ep_format_rx_res(struct rxm_ep *rxm_ep, const struct iovec *iov,
 			(*recv_entry)->rxm_iov.desc[i] = desc[i];
 	}
 
+	(*recv_entry)->multi_recv.len	= (*recv_entry)->total_len;
```

Comment:
Should we check for `(flags & FI_MULTI_RECV)` before this?

Reply:
Yes, it can be here, but it also ok to set len and buffer w/o the check (just to avoid if-statement from critical path).